### ISSUE TITLE
New roof

### ISSUE THREAD

**Issue by iteles** [dwyl/home#35]
We were told that it was highly likely that we would need to change the roof at `home` and following on from #20 (and the issues we have been having with increasing the energy provision for `home`), we have been seriously considering a Tesla Solar Roof.

https://www.tesla.com/pt_PT/energy
![image](https://user-images.githubusercontent.com/194400/50083638-50bd3e80-01ec-11e9-8076-05a09a6831bc.png)

However, two things have happened since:
+ Further investigation confirmed that the Solar Roof is still in its infancy and the timelines for this coming to Portugal are both unknown and 'not any time soon' https://www.cnbc.com/2018/09/24/tesla-solar-roof-tiles-where-are-they.html
+ We managed to get a very trust-worthy and more experienced roofer who has gone up to the roof and confirmed that with a little TLC, the roof tiles that are there will last "another 50 years" because they're great quality

Given that the most cost-efficient and environmentally-friendly option is to leave the existing tiles in place, this is what we will be going with. However, there will be a few tweaks to make sure we're looking after our story-filled shelter: insulating the roof for more energy efficiency (keeping the attic cooler in summer as well as the house warmer in winter) and protecting the roof tiles themselves.

## Dependencies on the New Roof
+ Placement of solar panels #20 
+ Placement of climatisation machines
  + This _can_ be done by removing a row of tiles from the existing roof but this may make the tiles  below the structure more  probe to falling off
+ Finalising the pool
  + This is semi-dependent but ideally the roof would  be completed because any falling roof tiles (according to the pool folks this is a very usual and probably occurrence) would damage the pool cover or lining
+ Placement of attic windows 
+ Finalising the plastering and painting of the ceiling in the attic
+ Finalising the flooring in the attic (a lot of rubble will come from opening the window holes so this would ideally not happen until the rood was completed

## Tasks
+ [x] Confirm decision on moving ahead with work on the roof
+ [x] Organise timelines for roofing work
+ [x] Remove all roof tiles
+ [x] Lay down roof structure
+ [x] Lay down paineis sandwich
+ [x] Open holes for windows
+ [x] Install windows 
+ [x] Finish off windows framing with peladur and insulation
+ [x] Build platform for external climatization machines
+ [x] Placement of solar panels
+ [x] Add guttering to roof #67

**Comment by iteles** [comment]
Have spoken to the roofer multiple times since we first spoke of this issue. He was to confirm timelines yesterday but will now have to confirm tomorrow.

The crux of the issue is: he's trying to finish his current piece of work but is having trouble because of the rain and morning ice and is also trying to push his next piece of work (a 4 month job) for a week and half to come and work with us.

Current hoped-for timeline is a 30th January start.

**Comment by iteles** [comment]
Turns out that Wednesday 30th January to the end of the week it will be raining so work can't start then. Update to come mid-next week.

**Comment by iteles** [comment]
We seem to be stuck at step 2 here for an age, as with everything! 

Spoke on Friday 2nd and with Northern Portugal on [red alert]((https://www.publico.pt/2019/02/01/sociedade/noticia/estradas-cortadas-voos-cancelados-arvores-caidas-portugal-enfrenta-tempestade-helena-1860334) due to Storm Helena, the roofers have been unable to finish their _previous_ job.

The weather _this_ week should **at least** allow them to finish their _current_ job and then 🤞 move onto our roof ***next week*** _weather permitting_ (11-Feb-2018).

![image](https://user-images.githubusercontent.com/4185328/52180761-4d086a00-27e2-11e9-8adc-8b5c0f52886e.png)

**Comment by iteles** [comment]
Bad news today.
Our roofer (highly recommended by family who had them do their roof 25 years ago and never had an issue) was trying to fit us in between two 4 month jobs and with the delays in the one before us because of the weather means that it will now clash with the next job which has to be started on a certain date next week contractually.
However, once one month on that job is complete, he can divert some of his manpower to come and work with us.

I have given him until the beginning of this coming week to provide us with a guarantee that the work will start no later than the 1st week of March 2019 (3 weeks from now).

### Impacts
+ **Attic windows** - these cannot be put in until the roof has been completed
+ **Flooring**: The attic flooring can only go in after the windows are complete, to avoid rubble being dragged over it and scratching it to pieces

**Comment by nelsonic** [comment]
@iteles this is _super_ lame. 😞 
Thanks for following up and updating the issue. 👍 
Do we have a timeline for the remaining windows to be complete?

> Did you make any progress with creating the Gantt Chart to visualise progress/dependencies?

**Comment by iteles** [comment]
Calls have been continuously ignored (4 per week for the last few weeks) or answered with 'I'm sorry, I don't have a date for you yet', so I have contacted another provider for a quote.

**Comment by iteles** [comment]
Meeting at site for quote from another provider on Friday (26-Apr-2019).

**Comment by iteles** [comment]
We've had another company come to visit the house for a quote but they can only start the work in September 😕 Finding more options today and tomorrow.

I've also added a list of dependent tasks to the top comment so we're aware of the limitations.

**Comment by nelsonic** [comment]
@iteles can we speak to Carlos about alternatives _today_?
_Specifically_ can we speak to him about:
+ [ ] carefully cutting open the holes for the Velux Windows
+ [ ] saving any tiles that can be re-used in the "storage" area of the attic
+ [ ] raising the tiles directly above and to the sides of the window and placing a _20_cm layer of [`roofmate`](https://www.dupont.com/products/ROOFMATEXPSInsulation.html) 
+ [ ] Installing the Velux windows to the _best_ of their abilities baring in mind that we will _pay_ them to return to the site _after_ the roof has been re-fitted to review (_and if necessary, repair_) the Velux installation.

The _current_ roof is not _leaking_ and our previous quote was to remove, clean and replace the _same_ tiles. 
![IMG_20190429_160904](https://user-images.githubusercontent.com/194400/57019013-332dc600-6c1d-11e9-8251-2afc7a6c6bb3.jpg)

I can do a _detailed_ technical drawing to illustrate what I'm _on_ about_ if you _agree_ that it's worth pursuing. The Velux windows chap was against it, but it was a "Computer Says NO!" without taking the time to _understand_ what I was proposing.

As we know from _experience_ the "experts" aren't _always_ right ...
![IMG_20190429_160843](https://user-images.githubusercontent.com/194400/57019223-c666fb80-6c1d-11e9-9f57-206e1cca61a0.jpg)

And _sometimes_ having an _outsider_ who can think _creatively_ about the problem _can_ lead to a solution that entrenched people would _never_ conceive of.

Each _day_ that the project is not _complete_ is costing us _significantly_ in terms of both out-of-pocket mortgage payments and lost revenue (["_opportunity cost_"](https://en.wikipedia.org/wiki/Opportunity_cost)) so it's worth our _cash_ to pay for the Velux guys to re-do the windows _entirely_ if it means we are open a month earlier.

**Comment by iteles** [comment]
> @iteles can we speak to Carlos about alternatives today?

We can't _today_ because it's a bank holiday and he won't pick up. But we can and should tomorrow, as was already the plan to speak to him in Braga for other things :+1:

**Comment by nelsonic** [comment]
@iteles should I attempt to do a 3D Rendering of what I'm trying to get them to do?

**Comment by iteles** [comment]
@nelsonic That might be useful, but I think the staircase #30 may still be higher priority so we can get it ordered. We'll speak to Carlos tomorrow anyway and get him speaking to people.

**Comment by nelsonic** [comment]
@iteles did the alt supplier give you a quote or just tell you "maybe September" ...?

**Comment by iteles** [comment]
@nelsonic "Maybe September but I can't say for sure" and 'We'll have a quote to you in 2 weeks'.

**Comment by nelsonic** [comment]
Two weeks ... 😞 
Surely, given the measurements of the building they have a form to calculate it in _minutes_... ⏳ 
Sometimes I'm tempted to stop writing code and become a construction worker. 👷 
I'm _sure_ I could get this work done in a couple of weeks from scratch using materials procured off OLX!

**Comment by iteles** [comment]
There's just a lot of demand right now so they're crazy busy.

**Comment by iteles** [comment]
After a number of budgets received, we have finally managed to find someone who can do the work in early June. I called immediately to schedule the windows installation and the earliest date for these in mid-July! All booked.

**Comment by iteles** [comment]
**We have a date!** 

> Work will start on the roof on 3rd June 2019

We had considered sourcing the roof tiles separately, however, the budget works out to being only €100 cheaper and I'd prefer to have everything taken care of by a single person who is then accountable for the work. So I made the executive decision to go for the slightly more expensive option and have the roofer take care of everything.

**Budget** to get the roof tiles removed, recycled, insulation added and new roof tiles laid is €6000.

**Comment by iteles** [comment]
OK, so there's been a delay already 🤦‍♀ 

> Roof works will now begin one week late, on Tuesday 11-Jun-2019 (Monday is a bank holiday)

Further updates: our **attic windows**, which had been booked for installation only on 15-Jul-2019, have now been rescheduled (after a few phone calls) to 21-Jun-2019 🎉

**Comment by iteles** [comment]
Aaaaand our roofers started dodging our calls and never showed up. Grand.

I've now organised with Pedro to get his men to come and do the work (they did the roofs at Craig's house) and Carlos Pinheiro to source the (same) materials for us.
We'll have a quote for the labour later today.

The only issue now is the window installation, so we'll see what the situation is on Monday. If it has to be moved, we're looking at end of July.

**Comment by iteles** [comment]
New roof details:

### Costs
+ Labour: €2850
+ Materials: €2900 for tiles + €500 for roofmate (this is an estimate on roofmate)
+ Clearing away old tiles (Carlos): €600

### Timelines
+ **Monday 24-Jun:** Work on the roof starts (this could have started on Friday 21-Jun but then the house would be roof-less over the weekend and it's threatening to rain)
+ **Friday 28-Jun:** Work is finishing and windows go in!

**Comment by nelsonic** [comment]
@iteles could you please ask Carlos/Pedro if the _labour_ would be cheaper if we used "Painel Sandwich"
on account of the fact that the panels are much larger and thus faster to install.

https://www.olx.pt/anuncio/mundiperfil-painel-sandwich-imitao-telha-preo-m2-IDtKMSh.html
![image](https://user-images.githubusercontent.com/194400/59743404-f028c000-9267-11e9-848c-6b2aa7a4006f.png)

The OLX listing indicates it's for this business: http://www.mundiperfil.pt/produto/munditelha

![image](https://user-images.githubusercontent.com/194400/59743682-b1473a00-9268-11e9-84e2-ab888716e096.png)

They are based in Barcelos and manufacture the Panels to the exact specification.
That means they would take the measurements and cut the panels at the factory and installation will be _much_ faster. (_like only a couple of days!_) They also make caleiros #67 the same color. (_so we could get a complete quote from them_)

![image](https://user-images.githubusercontent.com/194400/59743781-f9fef300-9268-11e9-8dcf-0608abcba055.png)



I don't mind making the phone calls / sending emails to the supplier to get quotes and timelines
+ Tel.: (+351) 253 834 545
+ mundiperfil@mundiperfil.pt

At **€17/m**<sup>2</sup> (_negotiable_) it will cost us _less_ in materials than the above quote for clay tiles.

**Comment by iteles** [comment]
Please do make the call. This is more expensive than the price you saw at Leroy Merlin though, right?

**Comment by nelsonic** [comment]
@iteles the price at Leroy was €14.59/m<sup>2</sup> but it's a different product:
![leroy-merlin-painel-sandwich-promo](https://user-images.githubusercontent.com/194400/59753961-20c72480-927d-11e9-9978-042810545c40.jpg)
The flat panels are usually cheaper.

The _cheapest_ price available online is €10.60/m<sup>2</sup>
https://www.olx.pt/anuncio/painel-chapa-sandwch-preo-qualidade-melhor-do-mercado-IDARzjH.html
![image](https://user-images.githubusercontent.com/194400/59754234-9cc16c80-927d-11e9-8a14-90b90307614c.png)

**Note**: that is for 450m<sup>2</sup> i.e €4,770 
(_we could ask Pedro/Carlos if they want the "surplus" at trade price ... 
e.g: if we only need 200m<sup>2</sup> for our roof, we they can have the rest at cost price ...
otherwise the price per m<sup>2</sup> will be higher ...  need to request a quote ..._)

**Question**: Did we ever get the measurements for the roof from the chap who measured it?

**Comment by iteles** [comment]
Yes, about 222m2 for the telhas was what we were quoted for 👍

**Comment by nelsonic** [comment]
222m x €17 = **€3,774**
€2900 for tiles + €500 roofmate = **€3,400**
@iteles Please ask Carlos `if` using Painel Sandwich will make the cost of _installation_ slightly cheaper.

> Note: "Painel Sandwich" = Structural Insulated Panels (**SIP**s) for anyone following along in English.

I've tried to read everything I can online about the pros/cons of telha tradicional vs. painéis sandwich ...
+ https://www.guiacasaeficiente.com/Paredes/PaineisSandwich.html
+ https://forumdacasa.com/discussion/44589/telhado-em-painel-sandwich-imitacao-de-telha-vantagens-e-desvantagens/
+ https://www.bautexsystems.com/blog/problems-with-structural-insulated-panels the problems described relate mostly to the panels that use OSB (ori­ent­ed strand boards) as one of the components. None of the suppliers in PT use OSB.

The main advantages appear to be thermal insulation (saving on energy bills) and speed of installation. 

The main advantage I was considering when I suggested the panels was the ease of installing the Solar Panels on top without breaking the tiles. 
The structure for the Solar Panels needs to be fixed to the roof bricks and holes would need to be drilled through the tiles (_or panels_) in order to fix the Solar Panels in place.

This is why I wanted to go past the Solar Panel supplier and get clarity on the best roofing options.
It does not compute in my mind that clay tiles would be a good idea if we need to drill through them for the Solar Panels ... But this _appears_ to be a reasonably "solved" problem considering the fact that _most_ roofs with solar panels in PT/EU use clay tiles and it hasn't stopped the proliferation of solar.

![image](https://user-images.githubusercontent.com/194400/59789197-63145400-92c5-11e9-817f-7cd97aebc8a8.png)

There really is nothing _conclusive_ on the durability of the SIPs vs. Clay tiles online. 🔍 😞 

We had agreed with Victor of the Solar Panels that when we started installing the `new` roof, we would call out their engineers to coordinate with the roofers and get the solar panel fixings installed at the same time to reduce the installation time & complexity of the solar panels.

**Comment by iteles** [comment]
@nelsonic On your last point, Vítor from the Solar Panels place will be at the house on Tuesday (before they start fully installing a roofing solution) to discuss options with the roofers 👍 I agreed this with them the second we got confirmation of a start date for the roof.

## On Costs
+ Apparently the €2,900 included the roofmate so that's the final cost of tiles
+ Onto the €3,774 of the painel sanduíche, we'd need to add the cost of the metal structure (which is an unknown at the moment but shouldn't be exorbitant)

The strangest thing for me at the moment is the idea that all 3 of the quotes we collected prior to trying to move ahead with clay tiles were upwards of €11,500.  
€3,775 (for the SIP) + €3,000 for the labour (roughly) + €600 (for clearing away the old roof tiles) = €7,375

It would be unlikely the metal structure would be €4,000!

A conservative estimate would be that at most, there is about a **€2000 difference** in pure costs between SIPs and clay tiles.

## On Labour Costs
The people we have working on the roof are pretty experienced with clay tiles and have estimated 1 week of labour for the tile roof.

Apparently, because of the metal structure, even though the roof itself will only take a day to lay, it will probably only be a day less of work and the labour costs will stay largely the same.

## On Timelines
Given our window situation (installation booked for Friday 28-Jun or 2-Aug if we miss the June deadline), the timeline is the biggest blocker here.

We're working with Pedro to find out if it would be possible to get the panels to us in time.

**Comment by iteles** [comment]
I have looked up the two things that most concerned me:
+ **Acoustic insulation:** I was wondering about rain hitting this material and it turns out that it's much of a muchness between SIPs and roof tiles
+ **Thermal insulation:** Given that we're already insulating the inside of the roof with lã de rocha and the outside with roofmate, it seems we still get a _small_ uptick in thermal insulation with SIPs but it's not significant at this stage

## Aesthetics
I , by _far_ prefer the one that imitates roof tiles and I'm starting to realise that our existing quotes were probably for this one, which is why they were more expensive.

You can _definitely_ see it from ground level  - the house across the road to ours has just been redone and you can clearly see a large portion of the roof tiles. Plus the side that is most visible won't have any solar panels to hide it.

We have gone to lengths to keep the story of the house and to keep some of the character of the fact that it's a renovated house. I feel like changing this to a no-tiled look would make a decent difference to the aesthetic of the house.

## For solar panels
The solar panels guys confirmed it would be much _easier_ for them to affix the solar panel structure on SIPs than on roof tiles (but our job isn't to make their jobs easier so I don't think this should be a consideration). There is of course a much lower risk of breaking any roof tiles in the process with SIPs which should be a consideration.

## Suppliers
Ferpainel: http://www.ferpainel.pt/pt/contactos/  
  + Para a semana vão produzir imitação de telha 400mm mas não têm possibilidade de entrega. Poderíamos carregar o material nós 5a ou 6a feira. 
+ Budgets: €16.25 + IVA = €19.98m2 for the ones imitating roof tiles
    + €11.25 + IVA = €13.83m2 for the flat panels 


Ondarev, can only install in September/October: http://www.ondarev.pt/pt/go/contactar-a-empresa


Não atenderam:
+ http://www.ftb.pt/en/Contactos Closed but open Monday
+ O Feliz https://ofelizpainel.com/contactos/
+ Alaço (encerrados para férias) https://www.alaco.pt/contactos/
+ http://www.irmalex.com/EN/index-4.html
+ Lopez painel https://lopezpanel.com/pt-pt/painel-sanduiche-braga/
+ Covema http://covema.pt/pt/contactos/
+ Perfitec Eng. Renato 913408632

https://www.painel2000.com/contactos/
253322188 - eng Angela Costa

**Comment by nelsonic** [comment]
@iteles thanks for doing this research (_making calls to suppliers_) ✅ 

When the building is covered in plants, I don't think anyone will pay much attention to the roof ...
![image](https://user-images.githubusercontent.com/194400/59915161-df1cb200-9413-11e9-841f-0e7852973ebb.png)

Should we investigate options for transporting the panels?

**Comment by iteles** [comment]
:confused: I still think it will make a big difference. 

I think the transport is the least of our worries. We can hire something if we need to. More of a question of making the decision to delay the windows and which roof type we prefer.

**Comment by iteles** [comment]
@nelsonic When do you want to go to the house this afternoon to take measurements?

**Comment by nelsonic** [comment]
In measuring (_deriving_) the area of the roof, we took accurate (_laser_) measurements of the house,
but (I) did not walk on top of the roof to get tape-measure dimensions from the _top_.
Instead, I decided to use trigonometry to calculate the area. 

I searched for "How to measure area of a roof" and found the following video:

[![image](https://user-images.githubusercontent.com/194400/60091638-01c50880-973d-11e9-821b-905131a6feaa.png)](https://youtu.be/0HZ1TYuRyPo)
https://youtu.be/0HZ1TYuRyPo

The _first_ instruction is to measure the pitch: (_obvs being an American tool it's in inches..._)
![image](https://user-images.githubusercontent.com/194400/60091819-5bc5ce00-973d-11e9-919b-2f2e12f91dbc.png)

Don't panic! _Thankfully_ a few seconds later there's a conversion table:
![image](https://user-images.githubusercontent.com/194400/60091799-52d4fc80-973d-11e9-8002-c4c8dcbe3867.png)

A quick search for "roof angle multiplier formula" got: https://roofonline.com/roof-slope-multiplier

Roof Pitch | (In Degrees) | Roof Pitch Multiplier
-- | -- | --
1/12 | 4.76° | 1.003
2/12 | 9.46° | 1.014
3/12 | 14.04° | 1.031
4/12 | 18.43° | 1.054
5/12 | 22.62° | 1.083
6/12 | 26.57° | 1.118

Created a new Google Spreadsheet (_other spreadsheets are available_) with the tabular data.
Made a graph using the data: (_the graph is useless because the Y-axis is too broad_)
![image](https://user-images.githubusercontent.com/194400/60092718-99c3f180-973f-11e9-86fd-3bde31b282ca.png)


![image](https://user-images.githubusercontent.com/194400/60092897-fc1cf200-973f-11e9-856f-647e69a444b5.png)

![image](https://user-images.githubusercontent.com/194400/60093599-bf51fa80-9741-11e9-807f-a74a20c4b41a.png)


I signed up for a free trial of SketchAndCalc which gave me 60 minutes to try the product ... 
Challenge accepted.

https://www.sketchandcalc.com/area-calculator
![image](https://user-images.githubusercontent.com/194400/60092483-0b4f7000-973f-11e9-8b92-bcd8f367d744.png)

![image](https://user-images.githubusercontent.com/194400/60093477-6c784300-9741-11e9-885a-8cdc739499d7.png)

SketchAndCalc gave us the (flat) area of: **194.89** m<sup>2</sup>
We applied the multiplier of 1.068 to get:  194.89 x 1.068 = **208.14 m**<sup>2</sup>


## Panels

In order to work out how many panels of which lengths are required, we drew them on and took the measurements.

Panels required for longest sides:
![image](https://user-images.githubusercontent.com/194400/60093599-bf51fa80-9741-11e9-807f-a74a20c4b41a.png)


Panels for the shorter sides:
![image](https://user-images.githubusercontent.com/194400/60095269-0f32c080-9746-11e9-8958-19c9e0d14c4a.png)

**Total metres of trim required: 38m**

![image](https://user-images.githubusercontent.com/194400/60095721-1d351100-9747-11e9-8050-d2e349cad09f.png)

Now to work out how many of each panel we need!

**Comment by nelsonic** [comment]
Roof divided into 1m strips (the SIPs panels):
![image](https://user-images.githubusercontent.com/194400/60125354-88043d80-9783-11e9-8691-703b1b69400d.png)

Panel cutting list:
![image](https://user-images.githubusercontent.com/194400/60125497-d0bbf680-9783-11e9-81d4-43589b11d6d3.png)

Sent to suppliers. ✅

**Comment by iteles** [comment]
After a lot of back and forth yesterday with confirmations, re-checking the order and chasing, this has now been confirmed from Irmalex for delivery Monday, at €15.50 / m<sup>2</sup> + IVA for the panels and €13 + IVA for the cumeeira. ✅

**Comment by nelsonic** [comment]
@iteles thanks for confirming. 🎉 
Can Irmalex supply the guttering #67 too? http://www.irmalex.com/PT/index-8.html
![image](https://user-images.githubusercontent.com/194400/60265229-27891380-98dd-11e9-9288-01dd266d55d5.png)
Or are you keen to have a _separate_ company be responsible for installing it?

**Comment by iteles** [comment]
Irmalex could _supply_ the guttering but then we'd have to have a different company (or Pedro's guys) install it.

I can ask for quotes.

**Comment by nelsonic** [comment]
@iteles if you already have a quote from a good supplier for the gutters #67, 
let's just get them to install them immediately after the roof is ready. 👍

**Comment by nelsonic** [comment]
Roof Panels Arrived today: (_thanks to @iteles tireless efforts to research, order and organise it!_) 🎉 
[![image](https://user-images.githubusercontent.com/194400/60462266-bcbf3b80-9c40-11e9-8924-bcf871b63c9c.png)](https://photos.app.goo.gl/6q5qetizFGMdgF277)
https://photos.app.goo.gl/6q5qetizFGMdgF277

📆 Installation tomorrow (_ETA: done by the end of the week_). 🤞

**Comment by iteles** [comment]
![IMG_1550](https://user-images.githubusercontent.com/4185328/60512899-bfb43d80-9ccd-11e9-9475-b61e48d20852.JPG)

Yesterday all the tiles were cleared from the roof.

The plan for today is to get the roof cleaned up and build out the metal structure that will lay underneath the roof panels. The distance between the metal struts must be _no wider than 1.5m_.
I tried telling the roofers that and they told me I was wrong (this is not unusual as yesterday there were two separate instances where I suggested something and was told 'no' only for them to do it when a man told them that actually it was a good idea). So I spoke to someone else at the building site they trust and to their boss, who both assured me it would be done.

Tomorrow, the panels start to go on, so we should print multiple copies of the panel drawing we did together last week to take to them in the morning: 
![image](https://user-images.githubusercontent.com/194400/60095814-684f2400-9747-11e9-93f7-339161018d4c.png)

![image](https://user-images.githubusercontent.com/194400/60095269-0f32c080-9746-11e9-8958-19c9e0d14c4a.png)

**Comment by iteles** [comment]
.. We are _part_ of the way there, but why would this be problem-free? 😄 
By this morning (9-Jul-2019), half of the roof was down, with one large and one small side to go.
The plan was to focus on the large side today so that the final two windows can get put in tomorrow as we only have the windows installation people today and tomorrow.

BUT the head roofer became ill and so they _all_ left after lunch with the roof unfinished.

Cue a panicked and very grouchy phone call from the windows installation guys.

I'm waiting for confirmation from Pedro (the roofers' boss) that we will all meet up at the house tomorrow for a 07h30 start with extra men where we can help to get the panels up to the roof and cut them if needs be.

**Comment by iteles** [comment]
This is done: 
![braga-home-solar-panels](https://user-images.githubusercontent.com/194400/64314612-85a23900-cfa7-11e9-94ad-c1cd6a93c99e.jpg)

We need a few finishing touches, such as the 'topos' but have moved these to: https://github.com/dwyl/home/issues/91

**Comment by nelsonic** [comment]
@iteles FYI the roof is not "done". It might _look_ done, but it's not finished.

**Comment by iteles** [comment]
@nelsonic Please add to the task list :+1:

**Comment by nelsonic** [comment]
@iteles I think we should have a _distinct_ issue "Roof Snagging List" that we cross-reference. 
Essentially there are 3 major snags:
1. The roofers put screws in the "valleys" of the roof tiles instead of only on the peaks. That means that they will be exposed to more rain water and thus will oxidise and the panels will get damaged. (the Solar Panel installer alerted us to this ... 🙄) So we need to apply 
"Tela líquida" (AKA "Membrana líquida") to all the screws to ensure that no water can enter.
This is the liquid rubber used by the Velux Windows installers to seal around the windows:
![braga-window-guy-applying-liquid-rubber](https://user-images.githubusercontent.com/194400/64414385-2584c300-d08b-11e9-8bd2-f538a7afb6be.jpg)

2. Almost all the Topos (ends) are rushed and thus poorly installed:
![braga-roof-topo-fail](https://user-images.githubusercontent.com/194400/64413845-3254e700-d08a-11e9-9efb-44a69bfd7554.jpg)
![braga-roof-topo-fail](https://user-images.githubusercontent.com/194400/64414453-464d1880-d08b-11e9-8e03-49a4d001cad4.jpg)
This means that we need a _competent_ person (_who isn't going to rush the work and run_) 
to carefully review each panel ending and where necessary remedy the join and paint with Membrana líquida to prevent water entering. 
https://www.leroymerlin.pt/Produtos/Construcao/Impermeabilizacao/Telhados-e-cobertura/WPR_REF_12281612
![image](https://user-images.githubusercontent.com/194400/64417206-8fec3200-d090-11e9-8d86-18a337de9d19.png)


3. **ANTS**!! 🐜 (_and other boring insects_) will _infest_ the ends of the Panels if not sealed properly.
(_we only learned about the pest problem **`after`** we had commissioned the roof in SIPs Panels ..._)
This is _related_ to `#2` above, but it is distinct in that the rain will not "climb" up from _underneath_. 
We need to apply an anti-bug treatment to the ends of _all_ the panels. For this I propose we use a Borate-based solution which will protect against a wide variety of boring bugs. 

_Much_ more detail on borates: http://www.americanborate.com/all-about-borates/borate-applications/borate-wood-preservation/
Bora-Care: https://youtu.be/Rmr_aEodyhg + https://youtu.be/lD-f9a8-OLw

It may seem counter-intuitive to use a ***wood*** treatment for SIPs Panels (_metal and foam_) 
but the foam in the SIPs panels is porous and will thus absorb the liquid so it's functionally similar.

Leroy has a product that will be appropriate: Tratamento universal AXTON incolor:
https://www.leroymerlin.pt/Produtos/Pintura-e-drogaria/Produtos-para-madeira-interior/Tratamento-contra-bichos-e-fungos/WPR_REF_16459800
![image](https://user-images.githubusercontent.com/194400/64416963-15bbad80-d090-11e9-9e89-e5dbd5f804f2.png)

I estimate that there is (_at least_) a **week** of work to be done on finishing the roof. 
The person doing the work will need to be competent/reliable and follow clear instructions.
They will (_obviously_) need to be comfortable with heights (_while wearing provided protection_).
Joaquim...? 💭

**Comment by iteles** [comment]
👍 The things people leave when they don't know you have a drone to inspect their work with closely 🤦‍♂ Moved to #92

**Comment by nelsonic** [comment]
Indeed. When your "toys" and flying skills come in handy to inspect the _work_... 😉 
Thanks for opening the new issue. ✅
